e2e/test_operator_upgrades/operator_upgrades_test.go

```diff
@@ -481,6 +568,12 @@ var _ = Describe("Operator Upgrades", Label("e2e"), func() {
 
 			// The cluster should still be able to upgrade.
 			Expect(fdbCluster.UpgradeCluster(targetVersion, true)).NotTo(HaveOccurred())
+
+			// NOTE: The coordinator process selected above is getting restarted - why the buggify option
```

Comment:
The coordinator process is not being restarted during the `kill` command (bounce phase) but in the later phase where all Pods are recreated. Assuming the coordinator is coming up in the incompatible connection list  this sub reconciler: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/controllers/remove_incompatible_processes.go will recreate the Pod.

Reply:
Ok, I will remove this comment then. Thanks!

Reply:
Done.